azurerm_app_service - support for the acr_use_managed_identity_creds and acr_user_managed_identity_id properties (#12745)
terraform-providers/terraform-provider-azurerm: website/docs/r/app_service.html.markdown

* `websockets_enabled` - (Optional) Should WebSockets be enabled?

> * `acr_use_managed_identity_creds` - (Optional) Are Managed Identity Credentials used for Azure Container Registry pull

Reviewer: ```suggestion
* `acr_use_managed_identity_credentials` - (Optional) Are Managed Identity Credentials used for Azure Container Registry pull
```

> * `acr_use_managed_identity_creds` - (Optional) Should Managed Identity Creds be used for Azure Container Registry pull
> 
> * `acr_user_managed_identity_id` - (Optional) If using User Managed Identity, the User Managed Identity Client Id

Reviewer: Reflecting the change of schema suggested above
```suggestion
* `acr_use_managed_identity_creds` - Are Managed Identity Credential used for Azure Container Registry.

* `acr_user_managed_identity_id` - (Optional) The User Managed Identity ID to use for accessing the Azure Container Registry.
```
